src/splatting/Splatting.js

```diff
@@ -1016,7 +1015,7 @@ async function main(initialFilePath) {
                     realityEditor.spatialCursor.gsToggleRaycast(false);
                     break FBORendering;
                 }
-                realityEditor.spatialCursor.gsToggleRaycast(true);
+                realityEditor.spatialCursor.gsToggleRaycast(false);
```

Comment:
Did you meant to test raycasting against the mesh model? Now it doesn't raycast on GS.

Reply:
Oh good catch, that was for a certain demo so the drawings would come out cleaner on the ground, but I'll undo this